maintenance/projects/js-themes-toolkit-v9-x/packages/generator-liferay-theme/lib/util.js

```diff
@@ -156,6 +156,14 @@ function sayHello(generator) {
 
 		https://github.com/liferay/liferay-frontend-projects/tree/master/projects/js-themes-toolkit#compatibility
 
+		`,
+		info`
+		For detailed information on version numbers of {unstyled}, {styled},
+		{classic}, {admin}, {frontend-css-common}, {@clayui/css} and {Bootstrap}
+		used by each release of Liferay DXP and Portal CE see the table at:
```

Comment:
> For detailed information on version numbers of [...] used by each release....

Version numbers of what? Perhaps should explicitly say `theme`. Same in the other file 👍

Reply:
OK, I'll change that to make it more explicit :+1:

Reply:
Done. [Here](https://github.com/liferay/liferay-frontend-projects/pull/527/files#diff-73cc2e2ee8288fbe7f8c6351f97fd9bac34ff6d0083a5fddec656c9bdde60358R159)'s the new text.